membership/README.md

```diff
@@ -46,3 +46,74 @@ The dues levels set by the Apereo Board for 2018 for educational organizations
 are as follows.
 
 TODO: add dues tables
+
+Core Foundation Dues and Sakai Supporting Subscription dues vary by the annual
+budget of the member institution.
+
+CAS and Bedework Supporting Subscription dues are a fixed price regardless of
+member institution budget.
+
+uPortal Supporting Subscription is offered at several levels not tied to
+member institution budget.
+
+### Dues in US dollars
+
+<table>
```

Comment:
:art: https://help.github.com/articles/organizing-information-with-tables/

Reply:
I suspect that a thoughtful Material treatment of this content wouldn't rely upon tables anyway, but point taken, and that Markdown syntax may very well be the next iterative step in wrangling this content.